release-tools/prow.sh

```diff
@@ -242,7 +242,9 @@ configvar CSI_PROW_DEP_VERSION v0.5.1 "golang dep version to be used for vendor
 # thus only makes sense in repos which provide their own CSI
 # driver. Repos can enable sanity testing by setting
 # CSI_PROW_TESTS_SANITY=sanity.
-configvar CSI_PROW_TESTS "unit parallel serial parallel-alpha serial-alpha sanity" "tests to run"
+# NOTE(xyang): Run only unit tests until snapshot beta API PR is merged.
+configvar CSI_PROW_TESTS "unit" "tests to run"
```

Comment:
instead of modifying this, I would modify https://github.com/kubernetes-csi/external-snapshotter/blob/master/.prow.sh instead to override this variable